pytas/models/projects.py

```diff
@@ -93,7 +94,45 @@ def remove_user(self, username):
         api = TASClient()
         return api.del_project_user(self.id, username)
 
+    @property
+    def active_allocations(self):
```

Comment:
I get an error when using these changes to view a project in portal while using master. It works if I use the portal's allocation branch. I think the issue is we try and set this property in the portal codebase but the property on the pytas side doesn't have a setter https://github.com/ChameleonCloud/portal/blob/master/projects/views.py#L162

Reply:
@zhenz this means we'll have to merge both this and the Portal PR at the same time, unless you change the requirements.txt of Portal to point to a pytas branch with these changes. They need to be rolled out at the same time.